Games/types/Mafia/roles/cards/BreadGiver.js

```diff
@@ -35,11 +35,17 @@ module.exports = class BreadGiver extends Card {
         this.listeners = {
             "rolesAssigned": function () {
                 for (let player of this.game.players){
-                    if(!player.hasItem("Bread")){
-                        player.holdItem("Bread");
+                    let items = player.items.map(a => a.name);
+                    let breadCount = 0;
+                    for (let item of items){
+                        if (item == "Bread")
+                            breadCount++;
+                    }
+                    while (breadCount < 2){
```

Comment:
this changes baker logic -- currently with 2 bakers everyone has 4 bread

Reply:
I thought with baker with old code each baker would have everyone start with 2 bread because with the second baker check it wouldn't give any more, but based on the role description i thought it was "if baker is in game, players start with 2 bread"